kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -137,9 +136,8 @@
       },
     },
     methods: {
-      genContentLink(id, kind) {
-        const routeName =
-          kind === ContentNodeKinds.TOPIC ? PageNames.TOPICS_TOPIC : PageNames.TOPICS_CONTENT;
+      genContentLink(id, isLeaf) {
+        const routeName = !isLeaf ? PageNames.TOPICS_TOPIC : PageNames.TOPICS_CONTENT;
```

Comment:
```suggestion
        const routeName = isLeaf ? PageNames.TOPICS_CONTENT : PageNames.TOPICS_TOPIC;
```
to make the `true` case first